Build fails in CentOS + deps compilation

```
CMake Error at /__w/Antares_Simulator/Antares_Simulator/_build/_deps/ortools-src/CMakeLists.txt:15 (cmake_minimum_required):
  CMake 3.18 or higher is required.  You are running version 3.17.5
```

Possible solution : download CMake from cmake.org, update the PATH accordingly.
```bash
echo "{path}" >> $GITHUB_PATH
````
See file **.github/workflows/centos7-system-deps-build.yml**

--- a/.github/workflows/centos7-system-deps-build.yml
+++ b/.github/workflows/centos7-system-deps-build.yml
@@ -32,9 +32,9 @@ jobs:
     - name: Install cmake
       run: |
            wget https://github.com/Kitware/CMake/releases/download/v3.25.3/cmake-3.25.3-linux-x86_64.tar.gz
-           tar xvf cmake-3.25.3-linux-x86_64.tar.gz
-           mv cmake-3.25.3-linux-x86_64.tar.gz /usr/local/cmake
-           echo "/usr/local/cmake" >> $GITHUB_PATH
+           tar xvf cmake-3.25.3-linux-x86_64.tar.gz -C /
+           echo "/cmake-3.25.3-linux-x86_64/bin" >> $GITHUB_PATH
+           rm cmake-3.25.3-linux-x86_64.tar.gz
 
     - name: Checkout
       run: |

--- a/.github/workflows/centos7-system-deps-build.yml
+++ b/.github/workflows/centos7-system-deps-build.yml
@@ -5,7 +5,6 @@ on:
     branches:
       - release/*
       - develop
-      - fix/centos-cmake
 
 jobs:
 
@@ -64,12 +63,12 @@ jobs:
     - name: Installer .rpm creation
       run: |
            cd _build
-           cpack3 -G RPM
+           cpack -G RPM
 
     - name: Installer .tar.gz creation
       run: |
            cd _build
-           cpack3 -G TGZ
+           cpack -G TGZ
 
     - name: Installer archive upload
       uses: actions/upload-artifact@v3

--- a/.github/workflows/centos7-system-deps-build.yml
+++ b/.github/workflows/centos7-system-deps-build.yml
@@ -5,6 +5,7 @@ on:
     branches:
       - release/*
       - develop
+      - fix/centos-cmake
 
 jobs:
 
@@ -24,10 +25,16 @@ jobs:
       run: |
            yum install -y epel-release
            yum install -y git redhat-lsb-core gcc gcc-c++ make wget centos-release-scl scl-utils rpm-build
-           yum install -y cmake3 devtoolset-9
+           yum install -y devtoolset-9
            yum install -y rh-git227-git
            yum install -y unzip libuuid-devel wxGTK3-devel boost-test boost-devel
 
+    - name: Install cmake
+      run: |
+           wget https://github.com/Kitware/CMake/releases/download/v3.25.3/cmake-3.25.3-linux-x86_64.tar.gz
+           tar xvf cmake-3.25.3-linux-x86_64.tar.gz
+           mv cmake-3.25.3-linux-x86_64.tar.gz /usr/local/cmake
+           echo "/usr/local/cmake" >> $GITHUB_PATH
 
     - name: Checkout
       run: |

--- a/.github/workflows/centos7-system-deps-build.yml
+++ b/.github/workflows/centos7-system-deps-build.yml
@@ -53,13 +53,13 @@ jobs:
            source /opt/rh/devtoolset-9/enable
            #git 2.x must be enabled for Coin compilation with CMake ExternalProject_Add
            source /opt/rh/rh-git227/enable
-           cmake3 -B _build -S src -DCMAKE_BUILD_TYPE=release -DBUILD_TESTING=ON
+           cmake -B _build -S src -DCMAKE_BUILD_TYPE=release -DBUILD_TESTING=ON
 
     - name: Build
       run: |
            source /opt/rh/devtoolset-9/enable
            source /opt/rh/rh-git227/enable
-           cmake3 --build _build --config release -j2
+           cmake --build _build --config release -j2
 
     - name: Installer .rpm creation
       run: |

